scripts/macos_setup_exiv2.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
korci przygotowanie osobnego repo, które wykonuje to transparentnie ;)

Reply:
Yyy czy ja wiem. Wydaje mi się, że takiego wroga warto trzymać blisko siebie, czyli w tym repo.
Inna sprawa, że mógłbym zrobić crate, który korzysta z tych binarek i tutaj mieć ten crate tylko jako dependency.
Pomyślę o tym :)

Reply:
tylko i wyłącznie myślałem o crate, którego brakuje ;)